Automattic/vip-go-mu-plugins
# validate_attachments: Bugfix and feature update

Bugfix:

* We were checking the database for the attachment GUID.  This is not the canonical URL for WordPress.  We should first check `wp_get_attachment_url()` which falls back to the GUID as a last resort.  We need this change because it's possible for the GUID to be incorrect, but the result of `wp_get_attachment_url()` is correct.

Updates:

* PHPCS fixes
* Added caching for found files.  This should increase speed when doing multiple rounds of validations.

---

## Description

A few fixes and updates to the validate-attachments CLI command.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `wp vip migration validate-attachments` on a site :)
